src/Engine/ProtoCore/Utils/CoreUtils.cs

```diff
@@ -670,6 +670,9 @@ public static string GetAssemblyFromClassName(ClassTable classTable, string clas
         /// <returns></returns>
         public static AssociativeNode CreateNodeFromString(string name)
         {
+            if (name == null)
```

Comment:
How about checking for `if string.IsNullOrEmpty(name)`?

Reply:
I have updated the code to use IsNullOrEmpty instead.